contracts/non-fungible-tokens/andromeda-cw721/src/testing/mod.rs

```diff
@@ -636,16 +474,39 @@ fn test_batch_mint() {
     }
 }
 
-#[test]
-fn test_batch_send_nft() {
+#[rstest]
+#[case::empty_batch(vec![], Some(ContractError::EmptyBatch {}), None)]
+#[case::unauthorized(
+    (0..5).map(|i| BatchSendMsg {
+        token_id: i.to_string(),
+        contract_addr: AndrAddr::from_string("contract_addr".to_string()),
+        msg: Binary::default(),
+    }).collect(),
+    Some(ContractError::Unauthorized {}),
+    None
+)]
+#[case::successful(
+    (0..5).map(|i| BatchSendMsg {
```

Comment:
I think this can just be written as a number in the test that mints the provided amount?